styles/NavBar.module.css

```diff
@@ -58,7 +58,6 @@
 .SideBar {
   list-style: none;
   width: 300px;
-  margin-top: 120px;
```

Comment:
why was this line removed? 
margin-top: 120px;

This is required for HomePage.